circuitbreaker_test.go

```diff
@@ -518,3 +518,30 @@ func TestPartialSecondBackoff(t *testing.T) {
 		t.Fatalf("expected breaker to be ready after more than nextBackoff time had passed")
 	}
 }
+
+func TestLogger(t *testing.T) {
```

Comment:
Is this test actually testing anything? I'd expect it to verify that it received the expected event(s).

Reply:
Oh man that was sloppy, I started then went and did something else and forgot I hadn't finished. Fixed now. Sorry for that.